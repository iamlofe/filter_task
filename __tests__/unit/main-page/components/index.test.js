import React from 'react';
import { shallow } from 'enzyme';

import IndexFilter from 'main/components';

describe('index component', () => {
    it('should shalow component without crashing', () => {
        const indexFilterC = <IndexFilter />;
        const indexFilterComponent = shallow(indexFilterC);
        expect(indexFilterComponent).toMatchSnapshot();
    });
});
