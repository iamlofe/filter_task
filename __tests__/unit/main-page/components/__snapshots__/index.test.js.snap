// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index component should shalow component without crashing 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Component />
</Provider>
`;
